### ISSUE TITLE
Songs in the playlist not playing at the scheduled time.

### ISSUE THREAD

**Issue by Mohammedwasimj** [AzuraCast/AzuraCast#6674]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

(2023-10-11 2:06) • Docker • PHP 8.2

### What happened?

Dear Team,

I am Using Azuracast "(2023-10-11 2:06) • Docker • PHP 8.2", but i facing problem in the scheduling the track. i scheduled the playlist 4 times a day and another playlist 1 time per day. (both playlist contains only one song)

so my requirement is to play the song once during the scheduled time. it should stop any other song which is playing and start playing the scheduled one. once played, then it continue the song which previously playing.

so in Playlist setting, i enabled both option 1) Loop Once (in the schedule tab) 2) Interrupt other songs to play at scheduled time. (in the Advanced Tab)

but even the song is not playing, means not queue up at their scheduled time.

so please help me to sort out this issue

**Comment by blindndangerous** [comment]
I'm having the same problem. I have two playlists, one containing regular music, one containing IDS for that show. I have both playlists scheduled to start and end at the same time and days but looking in the upcoming song queue, I don't see any IDs being pulled, and it doesn't show in the profile page under the scheduled heading.  Only the music playlist is being pulled from and my regular IDs are being used.

**Comment by caturria** [comment]
I'm not one of the devs, but "Interrupt other songs" is one of the advanced features that has to be handled directly by Liquidsoap instead of AzuraCast. You'll need to restart your station for a playlist scheduled this way to take effect.

**Comment by blindndangerous** [comment]
I actually got a bit further.  I had to delete my show ids playlist, remake it and reimport the files.  Once the scheduled time for it was set all looked good.  All matched and looking in the upcoming song queue they're loaded.  However, the normal IDS playlist is still showing up in that same queue unless I explicitly disable that playlist.  It does not have a schedule and none of my other playlists that have no schedule are doing this.
